### ISSUE TITLE
restarting pods throws error

### ISSUE THREAD

**Issue by ssinhas** [GoogleCloudPlatform/continuous-deployment-on-kubernetes#85]
delete and creating deployment doesn't work..  Throwing error

Failed to attach volume "jenkins-home" on node "gke-demo-default-pool-f7ab4024-6ttg" with: googleapi: Error 400: The disk resource 'projects/dev-lm/zones/us-central1-b/disks/jenkins-home' is already being used by gke-demo-default-pool-f7ab4024-npdg

**Comment by viglesiasce** [comment]
Were you able to get the pod to restart? It can take a couple of minutes for volumes to detach from the old node and reattach to a new one.

**Comment by ssinhas** [comment]
yup  I figure that one out...Question still remains how do you  automate this?
As it is part of Replication Controller, if pods dies RC recreate a pods and again it dies.

It goes in loop.

**Comment by viglesiasce** [comment]
This is automated, it should not loop. If it is looping forever then there is a bug.

**Comment by viglesiasce** [comment]
What version is your cluster?

**Comment by ssinhas** [comment]
1.6.4
